yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -659,12 +661,34 @@ def run(self, info):
 
         temp_filename = prepend_extension(filename, 'temp')
         self.to_screen('Embedding subtitles in "%s"' % filename)
-        self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
-        os.replace(temp_filename, filename)
+        if info['ext'] in self.AUDIO_EXTS:
+            self.embed_lyrics(input_files)
+        else:
+            self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
+            os.replace(temp_filename, filename)
 
         files_to_delete = [] if self._already_have_subtitle else sub_filenames
         return files_to_delete, info
 
+    def embed_lyrics(self, input_files):
+        audio_file = input_files[0]
+        subs = input_files[1]
+        if len(input_files) > 2:
+            self.report_warning('More than one subtitle file found. Only one will be embedded')
+        if not subs.endswith('.lrc'):
+            raise PostProcessingError('LRC subtitles required. Use "--convert-subs lrc" to convert')
+
+        with open(subs, 'r', encoding='utf-8') as f:
+            lyrics = f.read().strip()
+        if audio_file.endswith('.mp3'):
+            audio = mutagen.id3.ID3(audio_file)
+            audio.add(mutagen.id3.USLT(encoding=mutagen.id3.Encoding.UTF8, lang='und', desc='', text=lyrics))
```

Comment:
Are you sure only one lyrics can be added even if they have different language codes?

Reply:
![image](https://github.com/yt-dlp/yt-dlp/assets/89282645/231037cc-f818-497a-b41f-646d5a4be515)

language code from youtube and mutagen seems to be different